apitesters/paywalls.ts

```diff
@@ -1,9 +1,9 @@
 import RNPaywalls from "../react-native-purchases-ui/";
 
-async function checkpresentPaywall() {
-  await RNPaywalls.presentPaywall();
+async function checkPresentPaywall() {
+  const paywallResult: RNPaywalls.PAYWALL_RESULT = await RNPaywalls.presentPaywall();
```

Comment:
As you can see here, `PAYWALL_RESULT` right now is exported inside the `RNPaywalls` namespace. It feels that should be a different import IMO, but to do that, we should probably encapsulate the `presentPaywall` methods inside a class as we discussed. But that can come in a different PR.